squad/frontend/templates/squad/_threshold_table.jinja2

```diff
@@ -0,0 +1,95 @@
+<h2>
+    Threshold settings
+</h2>
+
+<div class='row'>
+    <div class='col-md-4'>
+        <a href="#" data-toggle="modal" data-target="#threshold_modal">
+            <button id="threshold_button" type="submit" class="btn btn-primary">Add threshold</button>
+        </a>
+    </div>
+</div>
+
+{% raw %}
+<div class='row row-bordered'>
+    <div class='col-md-1'>
+    </div>
+    <div class='col-md-5'>
+      <strong>Name</strong>
+    </div>
+    <div class='col-md-2'>
+      <strong>Higher is better</strong>
+    </div>
+    <div class='col-md-2'>
+      <strong>Value</strong>
+    </div>
+    <div class='col-md-2'>
+      <strong>Action</strong>
+    </div>
+</div>
+<div class='row row-bordered' ng-repeat="threshold in thresholds">
+    <div class='col-md-1'>
+        <i class='fa fa-line-chart'></i>
+    </div>
+    <div class='col-md-5'>
+        {{ threshold.name }}
+    </div>
+    <div class='col-md-2'>
+        <input type="checkbox" ng-if="!threshold.is_higher_better" disabled="disabled" />
+        <input type="checkbox" ng-if="threshold.is_higher_better" disabled="disabled" checked="checked" />
+    </div>
+    <div class='col-md-2'>
+        {{ threshold.value }}
+    </div>
+    <div class='col-md-2'>
+        <a href="#" data-toggle="modal" data-target="#threshold_modal" ng-click="setThreshold(threshold)" style="color: inherit; text-decoration: none;">
+            <i class='fa fa-pencil'></i>
+        </a>
+        <a href="#" ng-click="removeThreshold(threshold)" style="color: inherit; text-decoration: none;">
+            <i class='fa fa-trash'></i>
+        </a>
+    </div>
+</div>
+{% endraw %}
+
+<!-- Modal HTML -->
+<div id="threshold_modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title">Metric threshold</h4>
+      </div>
+      <form class="well" id="threshold_form" ng-submit="updateMetricThreshold()">
+        <div class="modal-body">
+          {% raw %}
+          <p ng-repeat="(name, errs) in errors" class="alert alert-danger"><strong>{{ name }}</strong>: {{ errs.join(', ') }}</p>
+          {% endraw %}
+
+          <div class="row form-group">
+            <label for="name" class="col-md-3 control-label">Metric name:</label>
+            <div class="col-md-10">
+              <input name="name" ng-model="currentThreshold.name" />
```

Comment:
maybe we should use select2 with connected search? There may be a lot of metrics available. Names of metrics can be queried from SuiteMetadata object like we do in single test comparison page. Can be fixed later though